fish-rust/src/builtins/type.rs

```diff
@@ -146,7 +144,7 @@ pub fn r#type(
                         streams.out.append(wgettext_fmt!(" (%ls)\n", comment));
                     }
                 } else if opts.get_type {
-                    streams.out.append(L!("function\n"));
+                    streams.out.appendln("function");
```

Comment:
(these are functional strings that probably shouldn't be translated - just like the scripting language itself isn't translated. we should keep that in mind for any translation guidelines)